README.md

```diff
@@ -13,6 +13,9 @@
 
 Rust bindings to the ECC's `zcash_script` c++ library.
 
+`zcash_script` links to `librustzcash`, which is written in Rust. Therefore,
+when updating `zcash_script`, we need to make sure that
```

Comment:
Incomplete docs?

Reply:
Thanks for catching that! I fixed this in 579c0c0a2 (I force-pushed this branch because the history became a mess)